player_bot/player_info/utils.py

```diff
@@ -39,8 +40,38 @@ def balls_lessons_payment(year: int, month: int, player: Player):
         tr_days_num_this_month: int = 0
         should_pay_balls: int = 0
 
-    should_pay_this_month = tr_days_num_this_month * tarif
-    return should_pay_this_month, should_pay_balls
+    pay_cancels = 0
+    cancels_count = 0
+
+    prev_month = month
+    if prev_month > 1:
+        prev_month = prev_month - 1
+
+    date = datetime.date.today().replace(month=prev_month, day=1)
+
+    cancel = Cancel.get_cancel_from_player(player, date)
+
+    if cancel is not None:
```

Comment:
```suggestion
    date = datetime.date.today().replace(month=prev_month, day=1)
    cancel = Cancel.get_cancel_from_player(player, date)
    if cancel is not None:
```
по смыслу все к одному и тому же относится, не нужно такие строки пробелами разделять